Tests/Testo/Auth/AuthServiceTest.php

```
@@ -0,0 +1,223 @@
+ * implementation, not arbitrary IdentityInterface identities), and the
+ * simple getIdentity()/isGuest() pass-throughs.
+ */
+#[Test]
+final class AuthServiceTest
+{
+    private function makeService(
+        CurrentUser $currentUser,
+        ?UserRepository $userR = null,
+        ?IdentityRepository $identityR = null,
+    ): AuthService {
+        $userR = $userR ?? m::mock(UserRepository::class);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Mockery' in method 'makeService'. (undefined)

(StaticAccess)